treeherder/etl/management/commands/pulse_listener.py

```diff
@@ -17,6 +17,9 @@ class Command(BaseCommand):
     help = "Read tasks and pushes from a set of pulse exchanges and queue for ingestion"
 
     def handle(self, *args, **options):
+        if env.bool('SKIP_INGESTION', default=False):
```

Comment:
Since we're moving the logic to `settings.py` then you will need to import `SKIP_INGESTION` from there instead of checking for the env variable.